idunn/api/utils.py

```diff
@@ -42,6 +50,68 @@ def fetch_es_poi(id, es) -> dict:
     result['properties'] = properties
     return result
 
+def fetch_bbox_places(bbox, es, indices, categories, max_size) -> list:
+    left, bot, right, top = bbox[0], bbox[1], bbox[2], bbox[3]
+
+    categories = re.findall(r'\((\w*?,\w*?)\)', categories)
+    classes, subclasses, class_sub = [], [], []
```

Comment:
`classes` contains the cases of the categories of the form `(<class>,_any)`
`subclasses` contains the cases of the categories of the form `(_any,<subclass>)`
`class_sub` contains the categories without any occurrence of `_any`